model/homework.py

```diff
@@ -78,6 +81,8 @@ def get_homework():
             'studentStatus':
             homework.student_status
             if user.role < 2 else homework.student_status.get(user.username),
+            'penalty':
+            homework.penalty if hasattr(homework, 'penalty') else None,
```

Comment:
`homework.penalty` 應該都會存在，可以不用檢查？
```suggestion
            homework.penalty
```

Reply:
我主要是怕已經存在的作業會沒有penalty這項值，所以做個檢查

Reply:
因為有指定預設值，所以若 DB 內沒有存的話，就會取預設值，我想都會存在？